portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -12,21 +12,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO: Modify servlet to access multiple messages from servlet.
+
 package com.google.sps.servlets;
 
+import java.util.*; 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns "Hello Jay Saleh!". TODO: modify this file to handle comments data */
+/** Servlet that writes an list of messages as a response. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  /** Hard coded messages used for tesing JSON */
+  private List<String> messages;
```

Comment:
Please prefer Immutable collections. With this you can do:
ImmutableList.of("","","")

private static final List<String> MESSAGES = ImmutableList.of(...)

Reply:
Having issues with `com.google.common.collect` was able to shorten length of `init()` by doing `messages = new ArrayList<>(Arrays.asList("one", "two", "three"));`

Done

Reply:
Oh what were the issues you ran into with it?

Reply:
I think it was an issue with the package management, in future steps I will be getting rid of this ArrayList. I am going to add this as a TODO if I have time later today.

Reply:
Sounds good. Please kindly add a todo comment.